src/proteomes/components/GenomeAssemblyAndAnnotationView.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
```

Comment:
I think you could use the `SimpleView` component instead of making a new one.

Reply:
I agree - updated. Also moved to the shared as it was being used outside of uniprotkb in a few places.